### DISCUSSION TITLE
turn off sorting in table

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#5490] [General]
Hi,

I am probably just going blind, but is there a way to turn off sorting (click on table column header)? In the cases where we have custom queries with pagination they can't really work (or am I missing something)?

M

**Comment by melohagan** [comment]
Hi @marco-santulli 

I don't think there is a way to do this. Maybe you could raise it here: https://github.com/Budibase/budibase/issues/3273
